integrations/standalone/tests/integration/parts/task.ts

```diff
@@ -179,11 +205,12 @@ class Task extends PartObject {
 }
 
 export class TaskTester extends NewPartTest {
-  constructor(options?: { name?: string; error?: RegExp }) {
-    super('Task', (part: Part) => new Task(part, options?.name ?? 'test name', options?.error ?? /f8/));
+  constructor(options?: { name?: string; error?: RegExp; persist?: boolean }) {
+    super('Task', (part: Part) => new Task(part, options?.name, options?.error, options?.persist));
   }
 }
 
 export const TaskTest = new TaskTester();
-
+export const StartRequestTaskTest = new TaskTester({ error: /EventAndGateway/, persist: true });
+export const TaskIntermediateTaskTest = new TaskTester({ error: /EventAndGateway/ });
```

Comment:
Seems not to be used in any tests yet?

Reply:
Ah yes. Forgot that. 
I thought it is easier if the test knows which parameters are needed for a specific element...